### ISSUE TITLE
mkdir and copy interpret rel URLs  differently

### ISSUE THREAD

**Issue by andre-merzky** [radical-cybertools/saga-python#457]
```
f = File('ssh://stampede/work/user/out.dat')
f.copy('/tmp/data/out.dat', CREATE_PARENTS)
```

that code is semantically *not* expressing a remote transfer, as the copy target is not an abs URL, so is interpreted in the f.url context, ie. on stampede.  That indeed works for the CREATE_PARENTS interpretation, but the file transfer will attempt to move the file to localhost...

**Comment by vivek-bala** [comment]
Hey Andre, is this issue address in any commit or PR?

**Comment by andre-merzky** [comment]
No, this is still open.  We work around this by always using full URLs, but that behavior is surprising (to say the least)  for unsuspecting users and needs fixing.

**Comment by vivek-bala** [comment]
For the example above, I think I would have expected the output to be copied to localhost and not on Stampede, FWIW :) But I mostly personally prefer full URLs wherever possible.

**Comment by andre-merzky** [comment]
> For the example above, I think I would have expected the output to be copied to localhost and not on Stampede, FWIW :)

Well, the holy SAGA spec says otherwise ... ;-)

**Comment by vivek-bala** [comment]
Alrighty! Leaving this ticket open in that case.
